src/mpi/coll/ibcast/ibcast.c

```diff
@@ -151,65 +155,83 @@ int MPIR_Ibcast_allcomm_auto(void *buffer, MPI_Aint count, MPI_Datatype datatype
 
     switch (cnt->id) {
         case MPII_CSEL_CONTAINER_TYPE__ALGORITHM__MPIR_Ibcast_intra_gentran_tree:
-            mpi_errno =
-                MPIR_Ibcast_intra_gentran_tree(buffer, count, datatype, root, comm_ptr,
-                                               cnt->u.ibcast.intra_gentran_tree.tree_type,
-                                               cnt->u.ibcast.intra_gentran_tree.k,
-                                               cnt->u.ibcast.intra_gentran_tree.chunk_size,
-                                               request);
+            MPII_GENTRAN_CREATE_SCHED_P();
+            mpi_errno = MPIR_TSP_Ibcast_sched_intra_tree(buffer, count, datatype, root, comm_ptr,
+                                                         cnt->u.ibcast.intra_gentran_tree.tree_type,
+                                                         cnt->u.ibcast.intra_gentran_tree.k,
+                                                         cnt->u.ibcast.
+                                                         intra_gentran_tree.chunk_size, *sched_p);
             break;
 
         case MPII_CSEL_CONTAINER_TYPE__ALGORITHM__MPIR_Ibcast_intra_gentran_scatterv_recexch_allgatherv:
-            mpi_errno =
-                MPIR_Ibcast_intra_gentran_scatterv_recexch_allgatherv(buffer, count, datatype, root,
-                                                                      comm_ptr,
-                                                                      cnt->u.
-                                                                      ibcast.intra_gentran_scatterv_recexch_allgatherv.scatterv_k,
-                                                                      cnt->u.
-                                                                      ibcast.intra_gentran_scatterv_recexch_allgatherv.allgatherv_k,
-                                                                      request);
+            MPII_GENTRAN_CREATE_SCHED_P
+                ();
```

Comment:
seems to be a accidental newline here.

Reply:
Yeah, due to misbehaving whitespace checker.